src/main/java/dev/jbang/cli/Init.java

```diff
@@ -177,7 +180,12 @@ String getRenderedScriptOrFile(Map<String, String> fileRefs, List<RefTarget> ref
 																															.getValue()))
 																.findFirst();
 			if (optionalRefTarget.isPresent()) {
-				return Util.getCwd().relativize(optionalRefTarget.get().to(outDir)).toFile().getPath();
+				Path path = optionalRefTarget.get().to(outDir);
+				if (absolute) {
+					return path.toFile().getPath();
```

Comment:
This can be simplified somewhat to `path.toString()`

Reply:
done